pkg/cloudprovider/providers/gce/gce_backendservice.go

```diff
@@ -44,6 +49,17 @@ func (gce *GCECloud) UpdateGlobalBackendService(bg *compute.BackendService) erro
 	return gce.waitForGlobalOp(op, mc)
 }
 
+// UpdateAlphaGlobalBackendService applies the given alpha BackendService as an update to an existing service.
+func (gce *GCECloud) UpdateAlphaGlobalBackendService(bg *computealpha.BackendService) error {
+	mc := newBackendServiceMetricContextWithVersion("alpha_update", "", computeAlphaVersion)
```

Comment:
Are we going with alpha_xxx or xxx_alpha or xxxalpha? Is there something there already? I want to be consistent...